sources/playground/README.md

```diff
@@ -0,0 +1,109 @@
+# CQL Playground
+
+Use the [`./playground.sh`](playground.sh) cli command to run examples or your own small experiments.
+
+Get some general help:
+
+```sh
+./playground.sh --help
```

Comment:
Can I recommend  `./play.sh` instead of `./playground.sh` -- it's already a lot more to type than `./go.sh`